packages/core-api/src/lib/loginPopup.ts

```diff
@@ -111,7 +119,12 @@ export function showLoginPopup(options: LoginPopupOptions): Promise<any> {
 
     const intervalId = setInterval(() => {
       if (popup.closed) {
-        const error = new Error('Login failed, popup was closed');
+        const errMessage = `Login failed, ${
+          targetOrigin !== window.location.origin
+            ? 'incorrect app origin'
```

Comment:
Shouldn't be a problem to show the expected origin here as it wouldn't be sensitive information. Also nothing that a regular user would see in a correctly configured app. Something like `Incorrect app origin, expected ${targetOrigin}`

Reply:
done as well! Thanks!